samples/sampler/stories/qa/PopupTabLayout.js

```diff
@@ -133,3 +152,75 @@ export const WithoutIcon = () => {
 
 WithoutIcon.storyName = 'without icon';
 
+export const WithVariousItems = () => {
+	const defaultOpen = true;
+	const [open, setOpenState] = useState(defaultOpen);
+	const toggleOpen = () => setOpenState(!open);
+	const handleClose = compose(toggleOpen, action('onClose'));
+
+	const [indexDisplay, setIndexDisplay] = useState(0);
+	const [indexSound, setIndexSound] = useState(0);
+
+	const handleDisplayNext = navNext(setIndexDisplay, indexDisplay, 'onNext');
+	const handleDisplayPrev = navPrev(setIndexDisplay, indexDisplay, 'onBack');
+	const handleSoundNext = navNext(setIndexSound, indexSound, 'onNext');
+	const handleSoundPrev = navPrev(setIndexSound, indexSound, 'onBack');
+
+	// Navigate menus with the right key. The left key is handled by framework.
+	const handleKeyDown = (setState, state) => (ev) => {
+		const {keyCode} = ev;
+
+		if (isRight(keyCode) && ev.target && !ev.target.hasAttribute('disabled')) {
+			navNext(setState, state, 'onNext')();
+		}
+	};
+
+	return (
+		<div>
+			<PopupTabLayout
+				open={open}
+				onClose={handleClose}
+			>
+				<Tab icon="picture" title="Display">
+					<TabPanels index={indexDisplay} onBack={handleDisplayPrev}>
+						<TabPanel>
+							<Header title="Display Settings" type="compact" />
+							<SwitchItem>Picture Modes</SwitchItem>
+							<Button size="small" onClick={handleDisplayNext} onKeyDown={handleKeyDown(setIndexDisplay, indexDisplay)}>button1</Button>
+							<Button size="small" onClick={handleDisplayNext} onKeyDown={handleKeyDown(setIndexDisplay, indexDisplay)} disabled>button2</Button>
```

Comment:
Could you reduce the width to verify spotlight behavior like FixedPopupLayout sampler?
![image](https://user-images.githubusercontent.com/34676178/118077548-43bdcd00-b3ef-11eb-840c-4caacb7e8bb0.png)